.circleci/config.yml

```diff
@@ -1,4 +1,6 @@
 version: 2.1
+orbs:
+  node: circleci/node@4.2.1
```

Comment:
https://circleci.com/developer/orbs/orb/circleci/node

This lets us easily install yarn and enables caching of yarn packages across subsequent builds on the same branch, where there are no changes for yarn between builds.